specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/rbac.json

```diff
@@ -0,0 +1,552 @@
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "SqlRoleDefinition_CreateOrUpdate",
+        "x-ms-examples": {
+          "CosmosDBSqlRoleDefinitionCreateUpdate": {
+            "$ref": "./examples/CosmosDBSqlRoleDefinitionCreateUpdate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "description": "Creates or updates an Azure Cosmos DB Role Definition.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/roleDefinitionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "requestBody",
+            "in": "body",
+            "required": true,
+            "description": "The properties required to create a new role definition.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleDefinitionProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Role Definition create or update request was accepted and will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleDefinitionResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "SqlRoleDefinition_Delete",
+        "x-ms-examples": {
+          "CosmosDBSqlRoleDefinitionDelete": {
+            "$ref": "./examples/CosmosDBSqlRoleDefinitionDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "description": "Deletes an existing Azure Cosmos DB Role Definition.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/roleDefinitionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The Role Definition was deleted.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleDefinitionResponse"
```

Comment:
In delete we do not return the object(because it is no more there). The response will be empty. Remove "schema" object.

Reply:
done